tsconfig.json

```diff
@@ -1,6 +1,12 @@
 {
   "compilerOptions": {
     "module": "commonjs",
+    "lib": [
+      "DOM",
+      "ES5",
+      "ScriptHost",
```

Comment:
Yes. It is mostly IE-specific extensions. It's not part of the ES spec. It is included when `--target: ES5` (and ES6) is used. See https://www.typescriptlang.org/docs/handbook/compiler-options.html under the `--lib` row.

> Note: If --lib is not specified a default library is injected. The default library injected is: 
► For --target ES5: DOM,ES5,ScriptHost

Anyway, I removed `ScriptHost` locally and it compiled. Ambient declarations from definitely typed don't typically reference it, so we should be OK removing it.